ramls/circulation.raml

```diff
@@ -406,6 +406,90 @@ resourceTypes:
             body:
               text/plain:
                 example: "Internal server error"
+    /notice-policy:
```

Comment:
Please update the `circulation` interface version (to 7.4) both here and in the module descriptor to reflect this addition.

Reply:
Done.